README.md

```diff
@@ -27,10 +27,14 @@ Edit [package.json](./client/package.json) and set `"proxy": "http://localhost:3
 ## How to run - using docker
 
 Install [docker](https://www.docker.com/docker-mac)
+NOTE: Config values for some container parameters (ie. db) are set up using [env files](https://docs.docker.com/compose/environment-variables/).
 
 From the root folder run:
 
 ```shell
+# Pull public postgres docker library
+docker pull postgres
+
```

Comment:
Interesting that I didn't need the above.

I ran `docker-compose up` and it pulled down the the pg images without me having to `docker pull postgres`

```
react-api-poc (postgres): $ docker-compose up
WARNING: The POSTGRES_PASSWORD variable is not set. Defaulting to a blank string.
Pulling db (postgres:9.6.5-alpine)...
9.6.5-alpine: Pulling from library/postgres
b1f00a6a160c: Pull complete
```

Did that error on your side before `docker pull postgres`?

Reply:
Weird yeah I was getting errors about not being able to pull from a public repo.  Finally when i tan the docker pull things started to work.